.gitignore

```diff
@@ -18,3 +18,5 @@ profile
 DerivedData
 
 local.properties
+
+.idea/*
```

Comment:
Can you remove this change? I believe user-specific files should be ignored in their global ;)